lib/constants/category.ts

```diff
@@ -3,6 +3,7 @@ export const Categories = [
   'ansible',
   'batect',
   'bazel',
+  'buildpacks',
```

Comment:
Remove this / it's not needed or helpful if there's only one manager in the category